src/components/my-app.js

```diff
@@ -60,6 +60,10 @@ class MyApp extends connect(store)(LitElement) {
         --app-drawer-selected-color: #78909C;
       }
 
+      app-drawer {
+        z-index: 1;
```

Comment:
Actually we have the same issue with app-header. We need to also ensure app-header being on top of the page content. Can you add `z-index: 1` to `app-header` and change `app-drawer` to have `z-index: 2`? I think this is what we did in Shop.

Reply:
Will do tonight.